document behavior if VCF(s) contain multiple

bw2 opened the pull request:
not sure if this is worth documenting here, but I had this question, and did some tests to figure out the answer.

tpoterba commented on docs/commands/importvcf.md:
```diff
@@ -22,7 +22,7 @@ $ hail importvcf /path/to/file.vcf.bgz write -o /path/to/output.vds
 
  - Hail makes certain assumptions about the genotype fields, see [Representation](#Representation).  On import, Hail filters (sets to no-call) any genotype that violates these assumptions.  Hail interpets the format fields: GT, AD, OD, DP, GQ, PL; all others are silently dropped.
-
+ - imporvcf does not preform deduplication - if the provided VCF(s) contain multiple records with the same chrom, pos, ref, alt, all these records will be imported and will not be collapsed into a single variant. 
```
typo "preform"

tpoterba commented:
@bw2 if you squash locally and force-push to your branch, this PR will update. No need to close/open a new one

bw2 commented:
Hey @tpoterba I tried, but am getting 
```remote: Permission to broadinstitute/hail.git denied to bw2.
fatal: unable to access 'https://github.com/broadinstitute/hail/': The requested URL returned error: 403```

tpoterba commented:
That's a bit weird.  You're pushing to the branch on your fork and still getting that error?

tpoterba commented:
I'm not sure.  @cseed @danking ideas?

bw2 commented:
Yes. I made the original changes + commits through the github UI.
Locally I tried doing

git fetch origin pull/805/head:add_importvcf_doc
git co add_importvcf_doc
git rebase -i HEAD~5
git push --set-upstream origin add_importvcf_doc




On Mon, Sep 19, 2016 at 2:14 PM, Tim Poterba <notifications@github.com>
wrote:

> I'm not sure. @cseed <https://github.com/cseed> @danking
> <https://github.com/danking> ideas?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hail-is/hail/pull/805#issuecomment-248075048>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AF83ql61njqrW-j2kg_KE_OWjY9l6sGMks5qrtEegaJpZM4KAD3V>
> .
>

danking commented:
@bw2 Could you paste the output of `git remote -v`? I suspect your `origin` is the broad institute origin. There's two problems with that:

 - we've moved the hail repository to hail-is/hail, and
 - you can only modify branches in your fork

So let's start by squashing your commits. Below I'll refer to your fork of hail as `myfork`.

```
git remote add myfork https://github.com/bw2/hail.git
git fetch myfork
git checkout patch-3
git rebase -i ecd1ed9^
git push myfork patch-3
```

When you return to this page, you should see just one commit listed.

As to your origin pointing to broad institute. If your origin looks like:

```
origin	https://github.com/broadinstitute/hail.git (fetch)
origin	https://github.com/broadinstitute/hail.git (push)
```

Then let's switch it to the new hail-is repository:

```
git remote remove origin
git remote add origin https://github.com/hail-is/hail.git
```

bw2 commented:
Daniel, you were right, my remote origin was `https://github.com/broadinstitute/hail.git`
so I changed it to https://github.com/hail-is/hail.git , but like you said, I don't have permissions to push directly to the main repo, so I changed it again to a fork - https://github.com/macarthur-lab/hail  and can submit a squashed PR from there.

bw2 commented:
BTW, this seems a bit inconsistent, but I can still click the 'edit' icons on github and create PR's directly on the main repo.

danking commented:
@bw2 Clicking the edit button on the `hail-is/hail` repo is definitely one of the supported interaction pathways. Each time you click it, you should get a [fresh branch under your hail fork](https://github.com/bw2/hail/branches). So, the particular branch that needs to be squashed is [bw2/hail](https://github.com/bw2/hail/tree/patch-3). Any new commits or force pushes to bw2/hail will automatically update this PR.

bw2 commented:
Thanks guys, I didn't realize the browser-based edits went into bw2/hail . Will get it right next time.

tpoterba commented:
the configuration changes we made mean that it's perfectly natural to handle this PR!

tpoterba commented:
Feel free to do the thing next time if it's at all easier 👍

bw2 commented:
Oh, nice!
